src/activities/ActivityUsageEntity.js

```diff
@@ -335,14 +423,39 @@ export class ActivityUsageEntity extends Entity {
 		return '';
 	}
 
-	_getDateSubEntity(dateClass) {
+	_getAvailabilityStartDateEntity() {
+		const availabilityDatesEntity = this._getSubEntityByClass(Classes.availabilityDates.availabilityDates);
+
+		if (!availabilityDatesEntity) {
+			return;
+		}
+
+		return availabilityDatesEntity.getSubEntityByClass(Classes.availabilityDates.startDate);
+	}
+
```

Comment:
This may need to change as well since having the dates entity doesn't guarantee taht you'll have the date startDate sub entity

Reply:
I guess this one is good depending on the expected behaviour:
- fad off would imply no dates entity which returns "undefined"
- fad on with no start date would also return "undefined"